packages/Webkul/Payment/src/Payment/CashOnDelivery.php

```diff
@@ -23,6 +24,11 @@ public function getRedirectUrl()
 
     }
 
+    public function isAvailable()
+    {
+        return (Cart::getCart()->haveStockableItems() && $this->getConfigData('active')) ? true : false;
```

Comment:
~~~php
return $this->cart->haveStockableItems() && $this->getConfigData('active');
~~~